corehq/apps/receiverwrapper/views.py

```diff
@@ -95,10 +95,12 @@ def _process_form(request, domain, app_id, user_id, authenticated,
 
     response = result.response
 
-    backend_tag = ('backend:sql' if should_use_sql_backend(domain) else
-                   'backend:couch')
     counter_name = 'commcare.xform_submissions.{}'.format(response.status_code)
```

Comment:
nit: might be more useful to have the status code as a tag as opposed to part of the metric